pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -758,9 +896,11 @@ func (vs *VSphere) HasClusterID() bool {
 // Notification handler when node is registered.
 func (vs *VSphere) NodeRegistered(node *v1.Node) {
 	glog.V(4).Infof("Node Registered: %+v", node)
+	vs.nodeManager.registerNode(node)
```

Comment:
Just for my clarification/review purpose, can you print the details of node and put is as comment.

Reply:
Here's the log for node:

{"log":"I1003 22:42:08.422980       1 vsphere.go:953] Node Registered: \u0026Node{ObjectMeta:k8s_io_apimachinery_pkg_apis_meta_v1.ObjectMeta{Name:kubernetes-node2,GenerateName:,Namespace:,SelfLink:/api/v1/nodes/kubernetes-node2,UID:ea375a76-a877-11e7-a2dd-005056b25dbc,ResourceVersion:3096,Generation:0,CreationTimestamp:2017-10-03 20:17:44 +0000 UTC,DeletionTimestamp:\u003cnil\u003e,DeletionGracePeriodSeconds:nil,Labels:map[string]string{beta.kubernetes.io/arch: amd64,beta.kubernetes.io/os: linux,kubernetes.io/hostname: kubernetes-node2,},Annotations:map[string]string{node.alpha.kubernetes.io/ttl: 0,volumes.kubernetes.io/controller-managed-attach-detach: true,},OwnerReferences:[],Finalizers:[],ClusterName:,Initializers:nil,},Spec:NodeSpec{PodCIDR:10.244.2.0/24,ExternalID:kubernetes-node2,ProviderID:,Unschedulable:false,Taints:[],ConfigSource:nil,},Status:NodeStatus{Capacity:ResourceList{cpu: {{2 0} {\u003cnil\u003e} 2 DecimalSI},memory: {{8369246208 0} {\u003cnil\u003e} 8173092Ki BinarySI},pods: {{110 0} {\u003cnil\u003e} 110 DecimalSI},},Allocatable:ResourceList{cpu: {{2 0} {\u003cnil\u003e} 2 DecimalSI},memory: {{8264388608 0} {\u003cnil\u003e} 8070692Ki BinarySI},pods: {{110 0} {\u003cnil\u003e} 110 DecimalSI},},Phase:,Conditions:[{OutOfDisk False 2017-10-03 22:42:06 +0000 UTC 2017-10-03 20:17:44 +0000 UTC KubeletHasSufficientDisk kubelet has sufficient disk space available} {MemoryPressure False 2017-10-03 22:42:06 +0000 UTC 2017-10-03 20:17:44 +0000 UTC KubeletHasSufficientMemory kubelet has sufficient memory available} {DiskPressure False 2017-10-03 22:42:06 +0000 UTC 2017-10-03 20:17:44 +0000 UTC KubeletHasNoDiskPressure kubelet has no disk pressure} {Ready True 2017-10-03 22:42:06 +0000 UTC 2017-10-03 20:17:44 +0000 UTC KubeletReady kubelet is posting ready status}],Addresses:[{InternalIP 10.161.83.64} {Hostname kubernetes-node2}],DaemonEndpoints:NodeDaemonEndpoints{KubeletEndpoint:DaemonEndpoint{Port:10250,},},NodeInfo:NodeSystemInfo{MachineID:833e0926ee21aed71ec075d726cbcfe0,SystemUUID:423206ED-283D-A740-8D6E-1E65D24CA6F3,BootID:68690535-c528-424e-93b1-1c01584dfe2e,KernelVersion:4.4.8-esx,OSImage:VMware Photon/Linux,ContainerRuntimeVersion:docker://1.11.0,KubeletVersion:v1.9.0-alpha.0.817+0e7cfe482272c9-dirty,KubeProxyVersion:v1.9.0-alpha.0.817+0e7cfe482272c9-dirty,OperatingSystem:linux,Architecture:amd64,},Images:[{[sandeeppissay/hyperkube-amd64:config-map-0.0.1] 501955286} {[gcr.io/google_containers/kubernetes-dashboard-amd64:v1.5.1] 103563297} {[gcr.io/google_containers/kubedns-amd64:1.7] 55064828} {[divyen/k8s-ignition:multivc-disabledvcp] 29620703} {[gcr.io/google_containers/exechealthz-amd64:1.2] 8374840} {[gcr.io/google_containers/kube-dnsmasq-amd64:1.3] 5125973} {[gcr.io/google_containers/pause-amd64:3.0] 746888}],VolumesInUse:[],VolumesAttached:[],},}\n","stream":"stderr","time":"2017-10-03T22:42:08.423333527Z"}